src/store/projects/project-create-actions.ts

```diff
@@ -63,3 +67,19 @@ export const createProject = (project: Partial<ProjectResource>) =>
             return undefined;
         }
     };
+
+export const addPropertyToCreateProjectForm = (data: ResourcePropertiesFormData) =>
+    (dispatch: Dispatch, getState: () => RootState, services: ServiceRepository) => {
+        const selector = formValueSelector(PROJECT_CREATE_FORM_NAME);
+        const properties = selector(getState(), 'properties') || {};
+        properties[data.key] = data.value;
+        dispatch(change(PROJECT_CREATE_FORM_NAME, 'properties', {...properties } ));
+    };
+
+export const removePropertyFromCreateProjectForm = (key: string) =>
+    (dispatch: Dispatch, getState: () => RootState, services: ServiceRepository) => {
+        const selector = formValueSelector(PROJECT_CREATE_FORM_NAME);
+        const properties = selector(getState(), 'properties');
+        delete properties[key];
+        dispatch(change(PROJECT_CREATE_FORM_NAME, 'properties', { ...properties } ));
```

Comment:
Same here, it's safer to modify copy (delete!) than modifying reference and then making a copy